cra-config.yaml

```diff
@@ -9,4 +9,5 @@ CRA_TARGETS:
       TF_VAR_kms_key_crn: "crn:v1:bluemix:public:hs-crypto:us-south:a/abac0df06b644a9cabc6e44f55b3880e:e6dce284-e80f-46e1-a3c1-830f7adff7a9:key:76170fae-4e0c-48c3-8ebe-326059ebb533"
       TF_VAR_prefix: "test-postgres-standard"
       TF_VAR_use_existing_resource_group: true
-      TF_VAR_resource_group_name: "geretain-test-postgres"
+      TF_VAR_resource_group_name: "Default"
```

Comment:
What is the requirement to update the resource group to "Default" here?

Reply:
Default resource group cannot be deleted, but the `geretain-test-postgres` can be - so just protecting it from breaking in the future

Reply:
I think Ill just set `use_existing_resource_group` to false instead so the plan will try to create one